src/main/java/LogicHandlers/CommandHandlers/Command.java

```diff
@@ -0,0 +1,11 @@
+package LogicHandlers.CommandHandlers;
+
+public interface Command {
+
+    /**
+     * Parses the content of the input, that succeeds the command.
+     *
+     * @param commandContent The content of the input.
+     */
+    void parseCommandContent(String commandContent);
```

Comment:
Probably unintended but I think you intend for this to be `public`? It's mostly for code consistency (interface methods are by default `public abstract`)